sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/util/polling/implementation/DefaultJsonSerializer.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.experimental.util.polling.implementation;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.azure.core.util.serializer.JsonSerializer;
+import com.azure.core.util.serializer.SerializerAdapter;
+import com.azure.core.util.serializer.SerializerEncoding;
+import com.azure.core.util.serializer.TypeReference;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.UncheckedIOException;
+
+/**
+ * Jackson based implementation of the {@link JsonSerializer}. This can be used as a default Json serializer if
+ * no JsonSerializerProvider is in the class path.
+ */
+public final class DefaultJsonSerializer implements JsonSerializer {
```

Comment:
We don't need to duplicate this from azure-core. Use `JsonSerializerProviders.createInstance(true)` to get the implementation in azure-core.